pyproject.toml

```diff
@@ -47,32 +47,39 @@ dependencies = [
 
 [project.optional-dependencies]
 dev = [
-    "autoflake",
-    "black",
-    "deepdiff",
-    "flake8",
-    "isort",
-    "jupyter",
+    # Documentation
     "mkdocs",
     "mkdocs-material",
     "markdown-include",
     "mkdocs-redirects",
     "mkdocstrings[python] >= 0.18",
+    "jupyter",
     "nbconvert",
+    # Lint
     "pre-commit",
-    "pytest",
+    "ruff",
```

Comment:
I like the grouping! 
Can you move ruff down within the group to match alphabetical order?

Reply:
Sure!